modules/create-wif-cluster-cli.adoc

```diff
@@ -179,13 +181,18 @@ $ ocm create cluster <cluster_name> \ <1>
 <5> Replace `<gcp_region>` with the {GCP} region where the new cluster will be deployed.
 <6> Optional: The subscription billing model for the cluster.
 <7> Optional: If you provided a value of `marketplace-gcp` for the `subscription-type` parameter, `marketplace-gcp-terms` must be equal to `true`.
-<8> Optional: The desired OpenShift version.
+<8> Optional: The desired {product-title} version.
 <9> Optional: Deploy to multiple data centers.
 <10> Optional: Enable autoscaling of compute nodes.
 <11> Optional: Minimum number of compute nodes.
 <12> Optional: Maximum number of compute nodes.
 <13> Optional: Secure Boot enables the use of Shielded VMs in the Google Cloud Platform.
 
+[IMPORTANT]
+====
+If an {product-title} version is specified, the version must also be supported by the assigned WIF configuration. If a version is specified that is not supported by the assigned WIF configuration, cluster creation will fail. If this occurs, upgrade the assigned WIF configuration to the desired version.
```

Comment:
If this occurs, **update** the assigned WIF configuration to the desired version.